File: tests/openmp_examples/program_control/dispatch.1.c
#pragma omp declare variant(foo_variant1) match(user={condition(foo_sub)})
#pragma omp declare variant(foo_variant2) match(construct={dispatch},user={condition(foo_sub)})
#pragma omp dispatch
#pragma omp dispatch
#pragma omp dispatch novariants(1)
#pragma omp dispatch nocontext(1)

File: tests/openmp_examples/program_control/dispatch.1.f90
!$omp      declare variant(foo_variant1) match(user={condition(foo_sub)})
!$omp      declare variant(foo_variant2) match(construct={dispatch},user={condition(foo_sub)})
!$omp   dispatch
!$omp   dispatch
!$omp   dispatch novariants(.true.)
!$omp   dispatch nocontext(.true.)
